phoenix5-spark/README.md

```diff
@@ -270,13 +333,13 @@ it falls back to using connection defined by hbase-site.xml.
 - `"jdbcUrl"` expects a full Phoenix JDBC URL, i.e. "jdbc:phoenix" or "jdbc:phoenix:zkHost:zkport",
 while `"zkUrl"` expects the ZK quorum only, i.e. "zkHost:zkPort"
 - If you want to use DataSourceV1, you can use source type `"org.apache.phoenix.spark"`
-instead of `"phoenix"`, however this is deprecated as of `connectors-1.0.0`.
-The `"org.apache.phoenix.spark"` datasource does not accept the `"jdbcUrl"` parameter,
-only `"zkUrl"`
+  instead of `"phoenix"`, however this is deprecated as of `connectors-1.0.0`.
```

Comment:
This was probably my mistake, but there has never been a connectors-1.0.0 release, and there will never be.
The next release will be 6.0.0.

I'd suggest simply not mentioning the version,